Notebooks/load_multiple_wells.ipynb

```diff
@@ -15,13 +15,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
     "# looks into root folder or subdirectories for LAS files\n",
     "asps = []\n",
-    "for root, dirs, files in os.walk(r'../data'):\n",
```

Comment:
@st4tic0 is there a reason you changed from relative to absolute file paths, this means the notebook won't be useable on other peoples computers. 

If there is problems with the directory maybe we could try 

```
import os

os.path.join('..', 'Data') # can't remember if this takes *args or list but hopefully you get the idea
```